docs/Coiote_DM_Device_Onboarding/Device_Provisioning.md

```diff
@@ -0,0 +1,85 @@
+# Automated Provisioning for Nordic boards
+
+## Introduction
+The device before it will be deployed in the field needs to be flashed and configured. This process takes some time and effort.
+
+This tutorial will show you how to provision Nordic boards using the provisioning script found [Anjay-zephyr-client repositorie] (https://github.com/AVSystem/Anjay-zephyr-client).
+
+## Prerequisites
+- Nordic board connected to your computer.
+- Installed [mcumgr command line tool](https://docs.zephyrproject.org/3.1.0/services/device_mgmt/mcumgr.html).
+- Zephyr development environment set up.
+- Registered account on [EU IOT CLOUD](https://eu.iot.avsystem.cloud).
+
+## Step 1: Getting Coiote Access Token
+The provision script can register the provisioned device to Coiote DM automaticlly. We will use this option in this tutorial but this is a optional step and can be omited. First an access token needs to be generated. 
+
+1. Create `get_token.sh` file:
+    ```
+    #!/bin/bash
+
+    if [ -z "$1" ]
+    then
+        echo "Usage:"
+        echo "$0 [USERNAME]"
+        exit 1
+    fi
+
+    USER="$1"
+    SERVER="https://eu.iot.avsystem.cloud"
+
+    read -p "Password for user $USER on $SERVER: " -s PASS
+
+    curl -X POST \
+       -H "Content-Type:application/x-www-form-urlencoded" \
+       --data-urlencode "grant_type=password" \
+       --data-urlencode "username=$USER" \
+       --data-urlencode "password=$PASS" \
+       "$SERVER/api/auth/oauth_password" 
+    
+    ```
```

Comment:
Would `anjay-zephyr-client`  in `tools/provisioning_tool` be ok?

Reply:
After some thought I would rather not add this script to the repo. Currently we are showing the user what is inside this script (he/she needs to paste it to a file). If I will add it to a repo this will become a blackbox that is asking for your credentials.